src/components/oculus-touch-controls.js

```diff
@@ -191,6 +192,18 @@ module.exports.Component = registerComponent('oculus-touch-controls', {
 
     // Offset pivot point
     controllerObject3D.position = PIVOT_OFFSET;
+
+    // Fix issue #2965 for default Touch model, rotate it 45 degrees.
+    // But if it isn't the default Touch model, don't.
+    controllerObject3D.traverse(function (o3d) {
+      if (o3d instanceof THREE.Mesh) {
+        if (o3d.name.startsWith('body_oculus-touch-controller-')) {
```

Comment:
It this check necessary. We're returning early above if `this.data.model === false`  (https://github.com/aframevr/aframe/pull/3511/files#diff-3056bccc97756be52ace666f5e213434R180)